default.nix

```diff
@@ -113,16 +111,6 @@ let
     utils = [(import ./overlays/utils)];
   };
 
-  rust-packages = rec {
-    overlays = [
-      (commonLib.pkgsDefault.callPackage ./overlays/rust/mozilla.nix {})
-      (import ./overlays/rust)
-    ];
-    pkgs = import sources."nixpkgs-19.09" {
-      inherit overlays config system crossSystem;
-    };
-  };
-
```

Comment:
We're using rust-packages with some projects utilizing naersk. This needs to stay. Ideally we should manage the mozilla overlay with flakes though.